libmamba/src/core/context.cpp

```diff
@@ -219,7 +219,7 @@ namespace mamba
         }
 
         std::map<std::string, AuthenticationInfo> res;
-        fs::u8path auth_loc(mamba::env::home_directory() / ".mamba" / "auth" / "authentication.json");
+        fs::u8path auth_loc(mamba::env::user_data_dir() / "auth" / "authentication.json");
```

Comment:
Yes, I'm not even sure this authentication support is properly (= fully) implemented and not sure anyone uses. So I agree, let's move it. The handful of people who use it can move their file manually